tests/ikev2-weak-dh/test.yaml

```diff
@@ -31,3 +32,38 @@ checks:
         ikev2.payload[1]: KeyExchange
         ikev2.payload[2]: SecurityAssociation
         ikev2.payload[3]: NoNextPayload
+
+  - filter:
+      count: 1
+      version: 5
```

Comment:
@inashivb can we do only one filter for versions 4 and 5 ?

Reply:
No, we could check for any version.